packages/fast-usdc-contract/test/supports.ts

```diff
@@ -189,55 +194,104 @@ export const commonSetup = async (t: ExecutionContext<any>) => {
 
   await registerKnownChains(agoricNamesAdmin, () => {});
 
-  // XXX: copied from orchestration supports
-  /** proxy for current sequence of the IBC Channel */
-  let ibcSequenceNonce = 0n;
   /**
-   * Simulate an inbound message to the vtransfer bridge.
-   *
-   * Uses the localchain bridge to lookup the last message and infer packet
-   * and transaction details.
+   * Find the Nth outgoing MsgTransfer and its sequence number from the localchain bridge log.
+   * @param index 0-based index from the start, or negative index from the end.
+   * @returns The MsgTransfer message and its sequence number.
+   * @throws If index is out of bounds or sequence is not found in the log result.
+   */
+  const getOutgoingTransfer = (index: number) => {
```

Comment:
Not at all necessary, but you might consider calling this `getOutgoingTransferAt` (or even `outgoingTransferAt`?) to rhyme with `Array.at`.